src/tests/ftest/util/data_mover_utils.py

```diff
@@ -0,0 +1,164 @@
+        # DAOS source container
+        self.daos_src_cont = FormattedParameter("--daos-src-cont {}")
+        # DAOS destination container
+        self.daos_dst_cont = FormattedParameter("--daos-dst-cont {}")
+        # Source service level
+        self.daos_src_svcl = FormattedParameter("--daos-src-svcl {}")
+        # Destination service level
+        self.daos_dst_svcl = FormattedParameter("--daos-dst-svcl {}")
+        # DAOS prefix for unified namespace path
+        self.daos_prefix = FormattedParameter("--daos-prefix {}")
+        # read source list from file
+        self.input_file = FormattedParameter("--input {}")
+        # work size per task in bytes (default 1MB)
+        self.chunksize = FormattedParameter("--chunksize {}")
+        # preserve permissions, ownership, timestamps, extended attributes
+        self.preserve = FormattedParameter("--preserve", False)
+        # use synchronous read/write calls (O_DIRECT)
+        self.direct = FormattedParameter("--direct", False)
+        # create sparse files when possible
+        self.sparse = FormattedParameter("--sparse", False)
+        # print progress every N seconds
+        self.progress = FormattedParameter("--progress {}")
+        # verbose output
+        self.verbose = FormattedParameter("--verbose", False)
+        # quiet output
+        self.quiet = FormattedParameter("--quiet", False)
+        # print help/usage
+        self.print_usage = FormattedParameter("--help", False)
+        # source path
+        self.src_path = BasicParameter(None)
+        # destination path
+        self.dest_path = BasicParameter(None)
+
+    def get_param_names(self):
+        """Overriding the original get_param_names"""
+
+        param_names = super(DataMoverCommand, self).get_param_names()
+
+        # move key=dest_path to the end
+        param_names.sort(key='dest_path'.__eq__)
+
+        return param_names
+
+    def set_datamover_params(self, src_pool=None, dst_pool=None, src_cont=None,
+                             dst_cont=None, display=True):
+        """Set the datamover params for the DAOS group, pool, and cont uuid.
+
+        Args:
+          src_pool(TestPool): source pool object
+          dst_pool(TestPool): destination pool object
+          src_cont(TestContainer): source container object
+          dst_cont(TestContainer): destination container object
+          display (bool, optional): print updated params. Defaults to True.
+        """
+
+        # set the obtained values
+        if src_pool:
+            self.daos_src_pool.update(src_pool.uuid,
+                                      "daos_src_pool" if display else None)
+            # setting src service level
+            src_svcl = ":".join(
+                [str(item) for item in [
+                    int(src_pool.pool.svc.rl_ranks[index])
+                    for index in range(src_pool.pool.svc.rl_nr)]])
+            self.daos_src_svcl.update(src_svcl, "svcl" if display else None)
```

Comment:
```suggestion
            self.daos_src_svcl.update(src_svcl, "src_svcl" if display else None)
```
Minor display change

Reply:
Done